src/org/labkey/targetedms/TargetedMSController.java

```diff
@@ -1057,8 +1059,8 @@ public Object execute(QCMetricOutliersForm form, BindException errors)
         private boolean _includeMR;
         private boolean _includeMeanCusum;
         private boolean _includeVariableCusum;
-        private String _startDate;
-        private String _endDate;
+        private java.sql.Date _startDate;
```

Comment:
Let's use java.util.Date instead - hopefully an easy swap

Reply:
not able to directly use java.util.Date in sql query.

Reply:
well no, it seems to be working, something was not right initially when I was trying to use it.

Reply:
replaced